src/manager/RestCountriesManager.ts

```diff
@@ -33,7 +55,18 @@ export class RestCountriesManager {
             throw new Error("unable to load language codes");
         }
         const allCountryCode: CountryResponse[] = await response.json();
-        for (const countryResponse of allCountryCode) {
+
+        // Remove english from countries where it is official, but not primary
+        const transformedCountryCode = allCountryCode.map(country => {
```

Comment:
this doesn't need to be a map, can just be a normal for loop. and to avoid 2 loops, it can be done in the loop below

Reply:
Removed redundant map